src/gsi/read_prepbufr.f90

```diff
@@ -1827,9 +1832,10 @@ subroutine read_prepbufr(nread,ndata,nodata,infile,obstype,lunout,twindin,sis,&
                  qm=sstq
               else if(gustob) then
                  gustqm=0
-                 if (kx==188 .or. kx==288 .or. kx==195 .or. kx==295 ) &
-                 call get_gustqm(kx,c_station_id,c_prvstg,c_sprvstg,gustqm)
-                 if ( l_rtma3d ) gustqm = 0  ! skipping get_gustqm for 3drtma run (missing list file)
+                 if ( i_gsdsfc_uselist/=1 .and. i_gsdsfc_uselist/=2 .and. .not.l_rtma3d)  then
+                    if (kx==188 .or. kx==288 .or. kx==195 .or. kx==295 ) &
+                    call get_gustqm(kx,c_station_id,c_prvstg,c_sprvstg,gustqm)
+                 endif
```

Comment:
Hi @ManuelPondeca-NOAA @MatthewMorris-NOAA ,
Just a comment: 
Based on the code in line 1835~1838, subroutine "get_gustqm" would be applied only for 2DRTMA/2DURMA run.  
Then, with Matthew's AutoQC running, will there be the gust obs use/reject list created for 3DRTMA run in the near future? so that the subroutine "get_gustqm" could be applied for 3DRTMA run?

Thanks,
-Gang